src/time/span.cr

```diff
@@ -286,6 +286,10 @@ struct Time::Span
   def self.zero
     new(0)
   end
+              
+  def zero?
+    self == typeof(self).zero
```

Comment:
Read @MakeNowJust's comment. This is a more general approach, comparing directly to the type's definition of "zero"